installers/release-notes.gradle

```diff
@@ -42,11 +42,11 @@ task releaseNotes {
 
     ByteArrayOutputStream gitContributorsOut = new ByteArrayOutputStream()
     exec {
-      commandLine "git", "log", "--no-merges", "--format=%cN", "16.6.0..${rootProject.gitRevision}"
+      commandLine "bash", '-c', "git shortlog -s --no-merges 16.6.0..${rootProject.gitRevision} | awk '!(\$1=\"\")' | tr '\n' ',' | sed s'/.\$//' | sed s'/^.//'"
```

Comment:
Instead of piping output, is it better to process `gitContributorsOut.toString()` using groovy instead? Lot more portable when running on OSes that don't have `awk` and `sed`.

```groovy
# remove the leading commit count
gitContributorsOut.toString().readLines().collect { line -> line.trim().split(/\s+/, 2).last() }
```